### ISSUE TITLE
CTRL+Click does not appear to work all the time now

### ISSUE THREAD

**Issue by Question2** [kblease/ProductionQueue#50]
Since updating to 0.14, ctrl+click no longer replaces the current build item all thet time. In effect, ctrl+click functions exactly as normal left click and adds the item to the end of the queue instead of replacing the current item.

**Comment by kblease** [comment]
Do you have a save file from when this is occurring?

Make sure that the item you're trying to force to the front of the queue doesn't rely on a prerequisite that is in the queue. In that case, ctrl+clicking the production item will put it as high up as it can.

**Comment by Question2** [comment]
I do but it happens on every save. CTRL+Click simply doesnt work on anything now. It doesnt work even if i do ctrl+click on a builder.

Right click to remove something from the queue doesnt work on anything either.

I could upload a save but most people in the past just say they dont have the mods or cant figure out how to load the save file without the mods...

**Comment by kblease** [comment]
Right-click to remove has been documented to no longer be possible, hence why that was replaced with the "drag to remove" feature. Firaxis changed their end of things in a way that broke it. If it is ever possible again in the future I will add that functionality.

I understand what you mean about sharing a save file. It can be difficult to share save files for that reason.

Unfortunately since I still cannot replicate any issues with ctrl+clicking, this is just going to have to stay open. Just to clarify, we are talking about ctrl+clicking on the production panel list, as that is the only place where ctrl+click has any specific behavior.

**Comment by Question2** [comment]
Im talking about ctrl+clicking on, say, a settler to replace the current item in the queue. Most of the time it does not do anything except add the item to the end of the queue. I can reproduce this in literally every game i play...

So for example i accidentally click on a settler. I want to replace the settler with a builder. I ctrl+click on the builder. It adds the builder to the end of the game. In order to replace the settler i have to add a builder to the end of the queue, drag it to the top of the queue then clear the queue. This is very time consuming.

To top it off, the city menu ALWAYS closes after i drag something to the top of the queue...so i have to click on the city again to clear the queue.

Sometimes when a city is producing nothing and i do a normal left click on, say, a builder, it will close the city menu. Sometimes when I click on a builder, it will keep the city menu open so that i can add more things to the queue. It doesnt matter whether the queue is open or closed, it just seems to happen randomly.

If you really want a save i just started a new game, founded a city on turn 1 and ctrl+click is not working already.

**Comment by Question2** [comment]
With the turn 1 save I noticed that if i do the following : 

1. Queue a settler
2. Ctrl+click on a builder
3. City menu closes
4. Builder is at the top of the queue

(The problem is i still need to open the city menu and clear the queue to remove the settler that I do not want to build)

But if i do this : 

1. Queue a settler
2. ctrl+click to queue a monument
3. The monument is added at the end of the queue

In the monument's case, middle click works and the monument is added to the top of the queue, but the city menu closes straight after and I still have to open the city to clear the queue.

I cant find a way to directly replace the current item...which is a huge hassle when I change my mind and want to build something else, because the production queue forces me to add something to the queue, click on the city again, then clear the queue manually.

**Comment by kblease** [comment]
> If you really want a save i just started a new game, founded a city on turn 1 and ctrl+click is not working already.

Similarly, every single game I start never has this problem, so it would seem it is an issue related to your specific situation. Do you have Rise and Fall installed/enabled? What is the version of your game? What is your operating system? Sorry if I've asked you those questions before on another platform.

The behavior of the panel automatically closing is a documented bug on a Windows 10 computer running Civ 6 1.0.0.229 (339082) without Rise and Fall, but I have not have an opportunity to diagnose that one. If you are also not running Rise and Fall, the ctrl+click issue is probably somewhat related. 

Very helpful and well-formatted as always, thank you.
